corehq/apps/app_manager/app_translations/validator.py

```diff
@@ -7,13 +7,16 @@
     get_unicode_dicts,
 )
 from corehq.apps.app_manager.app_translations.const import MODULES_AND_FORMS_SHEET_NAME
+from django.utils.translation import ugettext as _
+
 
 COLUMNS_TO_COMPARE = {
     'module_and_form': ['Type', 'sheet_name'],
     'module': ['case_property', 'list_or_detail'],
     'form': ['label'],
 }
-ROW_COUNT_MISMATCH_MESSAGE = "We found {} rows than expected. Validating what was uploaded."
+LESS_ROW_COUNT_MISMATCH_MESSAGE = _("We found less rows than expected. Validating what was uploaded.")
+MORE_ROW_COUNT_MISMATCH_MESSAGE = _("We found more rows than expected. Validating what was uploaded.")
```

Comment:
I don't think this'll work, since this is loaded at import time, and there isn't yet a translation context. Might have to use `ugettext_lazy` or define these in the body of the function somewhere.

Reply:
thanks, using `ugettext_lazy`